lib/types/applications/applications.go

```diff
@@ -46,6 +49,17 @@ type UtilityName string
 // UtilityFluid is the special utility name for the fluid token itself
 var UtilityFluid UtilityName = "FLUID"
 
+type ApplicationDataAmm struct {
```

Comment:
Should include that it's folded into ApplicationData as that's the container for this for now